build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+  tonic_build::compile_protos("proto/api.proto")?;
```

Comment:
It's better to rename our proto file to something like `network-api.proto` since the `proto` folder is shared between all Kudo services.

Reply:
Done !